scenariolib/event_click.go

```diff
@@ -34,7 +34,7 @@ func newClickEvent(e *JSONEvent) (*ClickEvent, error) {
 		return nil, errors.New("Parameter probability must be a number between 0 and 1 in a ClickEvent")
 	}
 
-	if docNo, validcast = e.Arguments["offset"].(float64); !validcast {
```

Comment:
actually yes. I had copied the click event as a template for my view event, so I was wondering why it was always sending doc rank=1